pkg/terminal/command.go

```diff
@@ -189,7 +189,10 @@ Current limitations:
 
 	clearall [<linespec>]
 
-If called with the linespec argument it will delete all the breakpoints matching the linespec. If linespec is omitted all breakpoints are deleted.`},
+	If called with the linespec argument it will delete all the breakpoints matching the linespec. If linespec is omitted all breakpoints are deleted.`},
```

Comment:
Remove the new tab (the idea is to have only the syntax indented so that it is rendered in monospace in markdown).